board/Ladder.java

```diff
@@ -0,0 +1,23 @@
+package board;
+
+public class Ladder {
+	private int start;
+    private int end;
+
+    public Ladder(int startval, int endval){
+        start = startval;
+        end = endval;
+    }
+
+    public int getstart(){
+        return start;
+    }
+
+    public int getend(){
+        return end;
+    }
+
+    public int elevate(){
```

Comment:
?

Reply:
didn't get ?

Reply:
It's doing the same thing as the getter. What's the point of having this function?